### DISCUSSION TITLE
Are you open to design contributions?

### DISCUSSION THREAD

**Discussion by fivenp** [AntlerVC/firetable#1002] [Support / Q&A] [answered]
Hey there!

As a long time Airtable user I made the switch to Rowy and enjoy it really really much.... Now through the nature of being a designer and somewhat an engineer loving OSS I was curious if you'd be open to design contributions?

Talking to a lot of other designers in the past there seems to be a huge interest in them contributing to OSS projects – they just struggle to understand how to do so.

Therefor I've built https://contribute.design/ which helps designers and open source software to understand each other better. Basically I'm trying to help projects get the guidelines for designers set up and create some visibility across designers for the project.

My question now is – would you be open to participate in this initiative and can I potentially support you in any way here? I'd really love to enable other designers to participate in amazing OSS projects....

**Comment by BeeBombshell** [comment, marked as answer]
Hi @fivenp, you can join our upcoming [office hours](https://discord.gg/vZbFQzje?event=1059820599184195604) and discuss your ideas with our team on Discord.
